packages/common/auth/src/types/authmaterial.ts

```diff
@@ -1,10 +1,28 @@
 import type { SDKExternalUser } from "./user";
 
+export type AuthMaterialBasic = {
+    jwtToken?: string;
+    refreshToken: string;
+};
+
+interface refreshTokenObject {
+    secret: string;
+    expiresAt: string;
+}
+
 export type AuthMaterial = {
     jwtToken: string;
-    refreshToken: {
-        secret: string;
-        expiresAt: string;
-    };
+    refreshToken: refreshTokenObject;
     user: SDKExternalUser;
 };
+
+export type AuthMaterialResponse = {
+    jwt: string;
+    refresh: refreshTokenObject;
```

Comment:
```suggestion
    refresh: RefreshToken;
```

Reply:
This has to remain `refresh` because it's the schema used in the backend which we can't change without making a breaking change.